src/Command/AllCommand.php

```diff
@@ -82,13 +92,9 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
             $tables = [$name];
         }
 
-        $commands = [
-            new ModelCommand(),
-            new ControllerCommand(),
-            new TemplateCommand(),
-        ];
-        foreach ($tables as $table) {
-            foreach ($commands as $command) {
+        foreach ($this->commands as $commandName) {
+            $command = new $commandName();
```

Comment:
Should we inline annotate here? So execute() below becomes clickable?

Reply:
Sounds good but I don't know how?

Reply:
`/** @var \Cake\Comand\Command $command */`